### PR TITLE
Fixed values for bleeds while bound and reduced cases tested

### PR DESCRIPTION
**Pull request by NightShadeI**
- Limited cases tested to bleeds, special bleeds or neither.
- Added multiplier in case you are walking target right when bind finishes and target is under bleed.
- Still requires overall testing.

### CODE REVIEW

File: Revolution Rotation Calculator.py [hunk 1]
```diff
@@ -9,13 +9,14 @@
...
 Bleeds = {'SHADOW TENDRILS': 1.8, 'FRAGMENTATION SHOT': 6, 'DEADSHOT': 6, 'CORRUPTION SHOT': 6, 'SMOKE TENDRILS': 5.4, 'CORRUPTION BLAST': 6, 'COMBUST': 6, 'SLAUGHTER': 6, 'DISMEMBER': 6, 'BLOOD TENDRILS': 4.8, 'MASSACRE': 6}  # time DOT abilties last (seconds)
-SpecialBleeds = ['MASSACRE','DEADSHOT'] # Bleeds that have their first hit affected by damage modifying abilities
-SpecialAbilities = ['DEEP IMPACT','SNIPE'] # Abilities that take longer than 1.8 seconds to use but will still have full impact from abilties in list CritBoost
+Walking_Bleeds = {'FRAGMENTATION SHOT': 1, 'COMBUST': 1, 'SLAUGHER': 1.5}
```
Comment (EZtouch) [inline Revolution Rotation Calculator.py]: This should be SLAUGHTER instead of SLAUGHER 🙂

**Review by EZtouch — CHANGES_REQUESTED** [review]
Fix typo

**Comment by NightShadeI** [comment]
Alright going to start with testing here too soon.

**Comment by Ditiae** [comment]
@NightShadeI Can you check out what I am planning on the other pull request? Could use your input on it. 😄

**Pull request merged**